kubelet/MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include requirements.in
```

Comment:
recent versions of distutils collect the readme automatically but we should be explicit, can you add `include README.md` here?